build.sbt

```diff
@@ -1,16 +1,16 @@
 scalaVersion := "2.12.10"
 
-val scalapbVersion = "0.10.8"
+lazy val scalapbVersion = "0.10.8"
```

Comment:
The `lazy` modifier here and below seems unnecessary. Are there any errors if it's removed?

Reply:
No, no errors, doing this just as a common practice, but it can stay without the `lazy` modifier